src/imex.rs

```diff
@@ -433,6 +433,7 @@ async fn import_backup_stream_inner<R: tokio::io::AsyncRead + Unpin>(
             .log_err(context)
             .ok();
     }
+    context.emit_event(EventType::AccountsItemChanged);
```

Comment:
I think this should be emitted right after `run_migrations()` above, but independently of the `run_migrations()` result because it's retried anyway on the next db open. But this isn't critical